src/modules/Transcript/test.tsx

```diff
@@ -7,7 +7,13 @@ import { Transcript, TranscriptUI } from '.';
 
 describe('TranscriptUI', () => {
     test('renders UI', () => {
-        render(<TranscriptUI paragraphs={[{ id: '1', sentences: [{ id: '1', text: 'TranscriptUI' }] }]} />);
+        render(
+            <TranscriptUI
+                toggleHeader={() => true}
+                headerExpanded={true}
```

Comment:
I've been moving away form testing the UI component directly now. I think a good test for this would be rendering the non-UI component and make sure you *don't* see any text from the expended header, then fire a click on the arrow and confirm that you then do see that text.

Reply:
this was just to prevent the test from breaking since I added more props - I didn't really add the functionality I would test yet (i.e. the actual event info)

Reply:
K. You could still pretty easily test checking for the proper arrow image right now, but can wait until more of the header is added too.